packages/browser/src/core/analytics/index.ts

```diff
@@ -421,7 +430,11 @@ export class Analytics
     return this._user.anonymousId(id)
   }
 
-  async queryString(query: string): Promise<Context[]> {
+  async queryString(query: string): Promise<Context[] | void> {
+    if (this.options.useQueryString === false) {
```

Comment:
Could  return empty array instead so the functions if doesn't change-- I think this would technically be a breaking change otherwise (certainly, it will break typescript types)

Reply:
Good catch! Let me fix that